webpack/assets/javascripts/react_app/components/common/Fill/FillReducer.js

```diff
@@ -12,7 +12,7 @@ export default (state = initialState, action) => {
       return state.setIn([payload.slotId, payload.fillId], payload.weight);
 
     case REMOVE_FILLED_COMPONENT:
-      return state.setIn([payload.slotId, payload.fillId], false);
+      return state.update(payload.slotId, x => x.without(payload.fillId));
```

Comment:
https://github.com/theforeman/foreman/pull/8755 fixes this reducer

Reply:
It has been merged, could you rebase pls?

Reply:
@ezr-ondrej, rebased  :)